proof-libs/fstar/core/Core.fst

```diff
@@ -4,3 +4,4 @@ include Rust_primitives
 include Core.Num
 include Core.Iter
 include Core.Ops
+include Core.Core_arch
```

Comment:
Same here, no include is required.
It's nice to `include` things only for definitions we deal with in a special way (e.g. the operator `+.` should always be in scope for every module)

```suggestion
```